src/devices/DCMotor/DCMotorWithStartStop.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+
+namespace Iot.Device.DCMotor
+{
+    /// <summary>
+    /// Direct current (DC) motor with Start/Stop
+    /// </summary>
+    public class DCMotorWithStartStop : DCMotor
```

Comment:
For the name, I'm not super sure. It feels like a bit long at the same time it describe swell what this does

Reply:
So maybe DCMotorStartStop?

Reply:
Let's see once @krwq or someone else will do a review.

Reply:
I'd rather have long descriptive name. The documentation is most important as I expect people to copy & paste samples